bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/obslog/ObslogGUI.java

```diff
@@ -625,6 +628,30 @@ public void setModel(TableModel model) {
                 }
             }
 
+            // Recalculates the width of the columns comment
+            void expandCommentColumn() {
+                TableColumn column = getColumnModel().getColumn(CommentTableModel.COL_COMMENT);
+                Component headerRenderer = getTableHeader().getDefaultRenderer()    .getTableCellRendererComponent(this, column.getHeaderValue(), false, false, -1, CommentTableModel.COL_COMMENT);
+
+                int preferredWidth = headerRenderer.getPreferredSize().width;
+                for (int i = 0; i < getModel().getRowCount(); i++) {
+                    final TableCellRenderer cellRenderer = this.getCellRenderer(i, CommentTableModel.COL_COMMENT);
+                    final Component c = table.prepareRenderer(cellRenderer, i, CommentTableModel.COL_COMMENT);
+                    final int width = c.getPreferredSize().width + table.getIntercellSpacing().width;
+                    preferredWidth = Math.max(preferredWidth, width);
+                }
+
+                // Give it a bit of space to breathe
+                int gutter = table.getFontMetrics(table.getFont()).charWidth('A');
+                preferredWidth += gutter;
+
+                // NB The order of these calls is critical to get the width correct
+                // DON'T reorder
+                column.setMinWidth(preferredWidth);
+                column.setMaxWidth(preferredWidth);
+                column.setPreferredWidth(preferredWidth);
```

Comment:
I know, you'd think you'd do `column.setWidth` but nope, you need the other ones **In the correct order**